bitbucket/resource_group.go

```diff
@@ -29,12 +37,22 @@ func resourceGroupCreate(d *schema.ResourceData, m interface{}) error {
 	client := m.(*BitbucketServerProvider).BitbucketClient
 
 	groupName := d.Get("name").(string)
+	importIfExists := d.Get("import_if_exists").(bool)
+	var newResource = true
 	_, err := client.Post(fmt.Sprintf("/rest/api/1.0/admin/groups?name=%s", url.QueryEscape(groupName)), nil)
 	if err != nil {
-		return err
+		if importIfExists && strings.Contains(err.Error(), "API Error: 409") {
```

Comment:
can you check the API status code instead of string?

Reply:
We've updated this to use the API status code.